traitsui/qt4/list_str_editor.py

```diff
@@ -512,6 +520,11 @@ def __init__(self, editor):
         # Configure context menu behavior
         self.setContextMenuPolicy(QtCore.Qt.CustomContextMenu)
 
+    def _dispose(self):
```

Comment:
i don't have enough context to know why `dispose` is private.

Reply:
The class is also private, so it really doesn't matter what we call it here.

Reply:
Except the instance is a public attribute on `ListStrEditor`. There are no clear guideline as to what is public to users of traitsui and what is protected. People import and subclass so freely so I'd prefer making things obviously private if we can. No one should be calling this except for the editor. Keeping it private for now.